cmd/manager/main.go

```diff
@@ -254,6 +255,15 @@ func main() {
 		}
 	}()
 
+	// ext authz server
+	authServer := authz.NewServer(logger)
+	go func() {
+		if err := authServer.ListenAndServe(":19000"); err != nil {
+			setupLog.Error(err, "Unable to start validation proxy")
```

Comment:
Aside from thinking that this shouldn't be done there.

Is this comment intentional? I would have thought it should be something like `authServer.ListenAndServe failed`.